TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -287,37 +296,70 @@ move(firstPhoto, from: "Summer Vacation", to: "Road Trip")
 In the example above,
 because the `move(_:from:to:)` function is synchronous,
 you guarantee that it can never contain possible suspension points.
+This encapsulates the code that temporarily makes the data model inconsistent,
+and makes it easier for anyone reading the code
+to recognize that no other code can run
+before data consistency is restored by completing the work.
 In the future,
 if you try to add concurrent code to this function,
 introducing a possible suspension point,
 you'll get compile-time error instead of introducing a bug.
 
-<!--
-  TODO you can also explicitly insert a suspension point
-  by calling ``Task.yield()``
-  https://developer.apple.com/documentation/swift/task/3814840-yield
--->
-
 <!--
   TODO add detail above about how the *compiler* can reason about
   the async/await version better too
   and give you better guarantees and clearer errors
```

Comment:
@ktoso Is there an example we can add here, or anything more detailed we should say?